Changes

```diff
@@ -41,6 +41,11 @@ OCaml 4.14 maintenance version
   (Gabriel Scherer, review by Nicolás Ojeda Bär and Vincent Laviron,
    report by Simon Cruanes)
 
+- #13516: Ensure runtime exceptions raised during initialisation (before
+  caml_global_data is loaded) are properly reported (regression of #5115 in
+  #11788)
+  (David Allsopp, review by ???)
```

Comment:
In my mind, Changes entries are for mainly meant for OCaml users rather than OCaml devs (which will read the Git log anyway). Accordingly, here's what I would find more useful from that point of view:
```
Fix bug whereby a segmentation fault could be triggered under certain error conditions during bytecode initialization.
Now the errors are properly reported. (Regression of ...)
```

Reply:
Very good point - how's the revised look?

Reply:
Looks good, thanks!